rpmspec.nanorc

```diff
@@ -2,11 +2,11 @@ syntax "Rpmspec" "\.spec$" "\.rpmspec$"
 
 color cyan  "\<(Icon|ExclusiveOs|ExcludeOs):"
 color cyan  "\<(BuildArch|BuildArchitectures|ExclusiveArch|ExcludeArch):"
-color cyan  "\<(Conflicts|Obsoletes|Provides|Requires|Requires\(.*\)|Enhances|Suggests|BuildConflicts|BuildRequires|Recommends|PreReq|Supplements):"
+color cyan  "\<(Conflicts|Obsoletes|Provides|Requires|Requires\(.*\)|Enhances|Suggests|BuildConflicts|BuildRequires|BuildRequires\(pre\)|Recommends|PreReq|Supplements):"
```

Comment:
Since you added `BuildRequires(pre)`, how about `BuildPreReq` ? 

https://www.altlinux.org/Spec#BuildRequires,_BuildPreReq,_BuildRequires(pre)

Reply:
Also, should it be better to change to `BuildRequires\(.*\)`, as in `Requires\(.*\)`?

Reply:
> Since you added `BuildRequires(pre)`, how about adding `BuildPreReq` ? as described here: https://www.altlinux.org/Spec#BuildRequires,_BuildPreReq,_BuildRequires(pre)

`BuildPreReq` is deprecated

> Also, should it be better to change to `BuildRequires\(.*\)`, as in `Requires\(.*\)`?

I don't know. I have never come across other uses of `BuildRequires()`
But I'll fix it. Maybe they will appear in the future.

Reply:
Oh okay, thanks for letting me know.

Reply:
> `BuildPreReq` is deprecated

Oop, guess not. But I really haven't seen anyone use it these days.